bin/git-fork

```diff
@@ -55,6 +90,12 @@ if [ "$origin" = true ]; then
 else
     # clone forked repo into current dir
     git clone "${remote_prefix}${user}/${project}.git" "$project"
+    until [[ -d "$project" ]]; do
+      echo "Github is being slow today. Waiting 10 secs to attempt clone."
```

Comment:
The issue I see there is that git clone is **very** verbose and just adds a lot of cruf to the terminal that does not need to be there (see below). Although I just realized that I should be sending stderr to /dev/null to avoid the excess cruf. 

Since the fork is asynchronous there is no way to determine when the fork is complete other than trying it. If I execute the fork command against github.com, the repo exists by the time that the clone is attempted. But if I try the fork command against the internal Github Enterprise instance, I end up waiting at least once for the fork to complete. (probably the fast LAN vs 200 ms across the net). 

The more that I think about it, the attempt to clone should probably only be attempted for about a minute or so. After that point the command should exit with an error code and a message that the repo could not be cloned locally. 

````
0:128 ᐅ git clone git@github.com:hickey/fioo.git foo  
Cloning into 'foo'...
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
````